sql/information_schema/information_schema.go

```diff
@@ -1238,7 +1238,7 @@ func getColumnNamesFromIndex(idx Index, table Table) []string {
 	for _, expr := range idx.Expressions() {
 		col := plan.GetColumnFromIndexExpr(expr, table)
 		if col != nil {
-			indexCols = append(indexCols, fmt.Sprintf("`%s`", col.Name))
+			indexCols = append(indexCols, fmt.Sprintf("%s", col.Name))
```

Comment:
Do you need the `fmt.Sprintf` here? Or can you just append `col.Name` directly as a string?

Reply:
btw... nice simplification here in this method to avoid unwrapping the backticks in multiple places in the code.